features/post/post-card/post-info-section/post-info-section.tsx

```diff
@@ -0,0 +1,114 @@
+import React, { ReactNode } from 'react';
+
+import { BookmarkOutline } from '@/assets/icons/bookmark-outline';
+import { HeartFilled } from '@/assets/icons/heart-filled';
+import { HeartOutlined } from '@/assets/icons/heart-outlined';
+import { PaperPlaneOutline } from '@/assets/icons/paper-plane-outline';
+import { DateFormatter, useDateFormat } from '@/shared/hooks/useDateFormat';
+import { useTranslation } from '@/shared/hooks/useTranslation';
+import { Avatar, IconButton, Typography } from '@/shared/ui';
+import { useRouter } from 'next/router';
+
+import s from './post-info.module.scss';
+
+type PostInfoProps = {
+  actions?: ReactNode;
+  createdAt: string;
+  icons?: {
+    avatar: string;
+    userName: string;
+  }[];
+  likesCount: number;
+};
+
+export const PostInfoSection = (props: PostInfoProps) => {
+  const { actions, createdAt, icons, likesCount } = props;
+  const { locale } = useRouter();
+  const { t } = useTranslation();
+
+  const customDateTimeFormatter: DateFormatter = parts => {
+    const date = new Date(createdAt);
+
+    return new Intl.DateTimeFormat(locale, {
+      day: 'numeric',
+      month: 'long',
+      year: 'numeric',
+    }).format(date);
+  };
+
+  const formattedDate = useDateFormat(
+    { day: 'numeric', month: 'long', year: 'numeric' },
+    customDateTimeFormatter
+  );
+
+  const formatNumberWithSpaces = (num: number | string): string => {
```

Comment:
A function with the same purpose already exists on the project. It is called `splitByThree` and is used in the `ProfileSummary` component. A loop is preferable to a regular expression for performance reasons (although it looks elegant).
You should put `splitByThree` in a separate file and use it where needed.